ui/package.json

```diff
@@ -25,11 +25,11 @@
     "ts-loader": "5.3.0",
     "ts-node": "7.0.1",
     "typescript": "3.1.4",
-    "webpack": "4.23.1",
-    "webpack-cli": "3.1.2"
+    "webpack": "^4.44.2",
```

Comment:
Could you pin the exact version like it's done for other dependencies? Makes things less prone to breakage as time passes.

Reply:
I did not notice, it's done. FYI I have seen versions that are not pinned in all the `package.json` files. I don't know if you want to pin everything or just some packages ?

Reply:
Everything :)